skills/csharp/experimental/itsmskill/Models/ServiceNow/ServiceNowNotification.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Newtonsoft.Json;
```

Comment:
License headers etc.

I am wondering what's the purpose of this update? 
For proactive demo, it has already been in enterprise VA.
If for ServiceNow, is it a standard structure? Where could I find documents about this? Thanks